boba/cli.py

```diff
@@ -4,8 +4,12 @@
 import click
 import os
 import subprocess
+from subprocess import PIPE
 from .parser import Parser
 from .output.csvmerger import CSVMerger
+import multiprocessing as mp
+import pandas as pd
+from shutil import copyfile
```

Comment:
Just in case you use any 3rd party libraries (i.e. not built-in), add them to the dependency list in https://github.com/uwdata/multiverse-spec/blob/master/setup.py

Reply:
Sure! These are all builtin, except for pandas, which is already in the dependency list.